TreeDB/documentservice/http_test.go

```
@@ -48,20 +48,16 @@ func TestHTTPMalformedJSONKeywordHybridAndErrorPayloads(t *testing.T) {
 		t.Fatalf("hybrid response=%+v", hybrid)
 	}
 
+	// v1alpha2: filters against undeclared meta fields keep failing closed,
+	// with a typed invalid_request naming the missing scalar schema entry.
 	req = httptest.NewRequest(http.MethodPost, "/v1/indexes/docs/search/keyword", bytes.NewBufferString(`{"query":"refund","top_k":1,"filter":{"field":"meta.repo","operator":"==","value":"gomap"}}`))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)